### PR TITLE
Add blog pages

### PR DESCRIPTION
**Pull request by juliendonck**
This adds the blog pages into the current set up through the makefile

### CODE REVIEW

**Comment by juliendonck** [comment]
@cloudhead not sure how all of this works (the Makefile stuff) so just tried to do it by copying what you did before. There's probably a better way to do it, you mind taking a look?

**Comment by juliendonck** [comment]
@abbey-titcomb @brandonhaslegs I've ported `towards decentralized ...` over and gave it basic styles. Let me know what you think. Also @abbey-titcomb what do you have in mind for a overview page? Are you expecting a place where you see all blogposts?

**Review by cloudhead — APPROVED** [review]

**Comment by abbey-titcomb** [comment]
@juliendonck great! so quick. 

I'm thinking for the overview we have something simple [like this](https://dev.mirror.xyz/). So each post has title, author (where we can distinguish between say, Monadic and Radicle Foundation), and date posted + have a preview of text, with a `Read More` button that leads to full post.

What do you think?

Probably will also need to add Blog to the Nav Header.

**Comment by juliendonck** [comment]
@abbey-titcomb the overview page is there. @brandonhaslegs wanna take a stab at creating a design for it? I can then implement it in code. Or you can do it straight in there if you want to.

File: docs/subscribed.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
   <title key="title">radicle</title>
   <link rel="shortcut icon" href="favicon.ico" />
-  <link rel="stylesheet" type="text/css" href="styles/style.css" />
+  <link rel="stylesheet" type="text/css" href="../styles/style.css" />
```
Comment (cloudhead) [inline docs/subscribed.html:6]: Why was this changed?

**Comment by cloudhead** [comment]
Looks good!

Couple of things I noticed:
* The first entry in `/blog` is hidden by the gif
* The text in the blog shows the regular arrow cursor when hovering on text, instead of showing the text cursor
* The article titles are not clickable
* There is no link to /blog in the menu bar
* I find the line width quite wide on the article page, ie. the words-per-line is very high
* I thought there was a cool diagram in the radicle link post that I don't see there
* The page titles for the articles is currently just "radicle"

**Comment by cloudhead** [comment]
* The "collaborating on radicle" title is cropped
* In that post, the list item line-height is set wrong (looks squeezed)

File: docs/subscribed.html (hunk 1 above)
Comment (juliendonck) [reply docs/subscribed.html:6]: cause the /blog is nested and /styles wouldn't resolve. Is there a better way to do it?

**Comment by juliendonck** [comment]
@cloudhead great comments. I'll clean up. I'm also expecting @brandonhaslegs to take a look and polish it at some point

**Comment by juliendonck** [comment]
Fixed:
- [x] The first entry in /blog is hidden by the gif
- [x] The text in the blog shows the regular arrow cursor when hovering on text, instead of showing the text cursor
- [x] The article titles are not clickable
- [x] There is no link to /blog in the menu bar
- [x] I find the line width quite wide on the article page, ie. the words-per-line is very high
- [x] In that post, the list item line-height is set wrong (looks squeezed)
- [x] I thought there was a cool diagram in the radicle link post that I don't see there
- [x] The page titles for the articles is currently just "radicle"
- [x] The "collaborating on radicle" title is cropped

File: docs/subscribed.html (hunk 1 above)
Comment (cloudhead) [reply docs/subscribed.html:6]: But this is the subscribed page

**Comment by SNikhill** [comment]
Greetings,

I am sorry to interrupt. I was just looking at the code. And wanted to suggest that if dynamic titles is what you guys want, you can change it using `document.title`.

Like, create a script that waits for the page to load completely. Then, make a function that Picks up the title for the Blog using the top-most `h1` or `h2` tag or you may give a `title` class and may target the element using the same.
Then, the text content from the same could be assigned to the `document.title`.

I shall be able to assist in the same.

Regards
SNikhill

**Comment by cloudhead** [comment]
> Greetings,
> 
> I am sorry to interrupt. I was just looking at the code. And wanted to suggest that if dynamic titles is what you guys want, you can change it using `document.title`.
> 
> Like, create a script that waits for the page to load completely. Then, make a function that Picks up the title for the Blog using the top-most `h1` or `h2` tag or you may give a `title` class and may target the element using the same.
> Then, the text content from the same could be assigned to the `document.title`.
> 
> I shall be able to assist in the same.
> 
> Regards
> SNikhill

This won't work, as the HTML will be wrong and thus web crawlers will not see the correct title.

**Comment by SNikhill** [comment]
> > Like, create a script that waits for the page to load completely. Then, make a function that Picks up the title for the Blog using the top-most `h1` or `h2` tag or you may give a `title` class and may target the element using the same.
> > Then, the text content from the same could be assigned to the `document.title`.
> 
> This won't work, as the HTML will be wrong and thus web crawlers will not see the correct title.

I see, so basically, since the title mentioned in the HTML will be incorrect and if someone was to scrape the page then, they will see the title mentioned in the code, which in this case will be incorrect, as the actual title will be loaded later using JS.

Well, thank you for pointing that out. :slightly_smiling_face:   I never really gave it a thought.

**Comment by cloudhead** [comment]
> I see, so basically, since the title mentioned in the HTML will be incorrect and if someone was to scrape the page then, they will see the title mentioned in the code, which in this case will be incorrect, as the actual title will be loaded later using JS.

That's right, and hence search results will show the incorrect title.

**Comment by cloudhead** [comment]
Looks great! The only thing I'm still finding is this one:

> * [x]  In that post, the list item line-height is set wrong (looks squeezed)
> *

2021-02-05-125808_993x385_scrot

Otherwise, everything seems fixed.

**Comment by juliendonck** [comment]
> > * [x]   In that post, the list item line-height is set wrong (looks squeezed)
> 
> 2021-02-05-125808_993x385_scrot
> 
@cloudhead Fixed

**Review by cloudhead — APPROVED** [review]
